packages/compass-components/src/components/index-keys.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import Badge, { Variant } from '@leafygreen-ui/badge';
+import { css } from '@leafygreen-ui/emotion';
+import Icon from '@leafygreen-ui/icon';
+import { spacing } from '@leafygreen-ui/tokens';
+
+type IndexDirection =
+  | -1
+  | 1
+  | '2d'
+  | '2dsphere'
+  | 'text'
+  | 'geoHaystack'
+  | number;
+
+type IndexKeysProps = {
+  keys: {
+    field: string;
+    value: IndexDirection;
+  }[];
+};
+
+const containerStyles = css({
+  display: 'flex',
+  gap: spacing[1],
+});
+
+const IndexDirectionIcon = ({ direction }: { direction: IndexDirection }) => {
+  return direction === 1 ? (
+    <Icon glyph="ArrowUp" />
+  ) : direction === -1 ? (
+    <Icon glyph="ArrowDown" />
+  ) : (
+    <>({String(direction)})</>
+  );
+};
```

Comment:
Without a proper label / title, VO in browser will read this as "Arrow Up Icon" (default title that lg icon will have) which is really not helpful if you are using assistive tech:

![image (5)](https://user-images.githubusercontent.com/5036933/182642856-20ceece2-8ab1-425e-9357-3500ebaef656.png)

At the very least let's add a proper label here. Even better would be to use aria-hidden in combination with VisuallyHidden label to provide a meaningful description for this badge (or the whole cell even), something like this:

![image (6)](https://user-images.githubusercontent.com/5036933/182642984-3794c0fe-3326-4612-90a7-da93876254d2.png)